qrisk/tests/test_stats.py

```diff
@@ -2,75 +2,154 @@
 
```

Comment:
I don't think that is sufficient justification for continuing to use an unsupported library. The first sentence of the nose docs is:

> Nose has been in maintenance mode for the past several years and will likely cease without a new person/team to take over maintainership. New projects should consider using Nose2, py.test, or just plain unittest/unittest2.